### PR TITLE
PMM-7733 Percona platform account

### PR DESCRIPTION
**Pull request by yurkovychv**

### CODE REVIEW

File: tests/pages/perconaPlatform.js [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+
+    // Verify validation message for "email@" value
+    I.appendField(this.fields.emailField, '@');
+    I.seeTextEquals(this.messages.invalidEmail, this.elements.emailValidation);
+
+    // Verify validation message for "email@domain#.com" value
+    I.appendField(this.fields.emailField, 'domain#.com');
+    I.seeTextEquals(this.messages.invalidEmail, this.elements.emailValidation);
+
+    // Verify there is no validation for "email@domain.com" value
```
Comment (atymchuk) [inline tests/pages/perconaPlatform.js:116]:
```suggestion
    // Verify there is no validation error for "email@domain.com" value
```

File: tests/platform_test.js [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+    });
+
+    // Verify Sign Up button is disabled and Back to Login button is present and enabled
+    I.seeAttributesOnElements(buttons.signUp, { disabled: true });
+    I.seeTextEquals('Back to login', buttons.backToSignIn);
+    I.seeAttributesOnElements(buttons.backToSignIn, { disabled: null });
+
+    // Focus on Email, First Name, Last Name fields and terms checkbox to verify that fields are required
+    I.usePlaywrightTo('focus on email and password fields', async ({ page }) => {
+      page.focus('[data-qa="email-text-input"]');
```
Comment (atymchuk) [inline tests/platform_test.js:88]: FYI: you may want to use a function to save a few keystrokes :)
https://github.com/percona-platform/core-ui/blob/8f206e9ac5a8693b8bec60f65f5f1bdf2d3fcf54/src/shared/utils/test.utils.tsx#L4

**Review by atymchuk — DISMISSED** [review]

File: tests/platform_test.js [hunk 3]
```diff
@@ -0,0 +1,175 @@
...
+);
+
+Scenario(
+  'PMM-T399 Verify user is able to Sign Up @platform @settings',
+  async ({ I }) => {
+    I.click(buttons.goToSignUp);
+    I.waitForVisible(elements.signUpForm, 30);
+
+    const newUserEmail = await I.generateNewEmail();
+    const newUserPassword = 'MySuperSecretTempPassword123321';
```
Comment (puneet0191) [inline tests/platform_test.js:155]: could use faker too here?

**Review by puneet0191 — APPROVED** [review]

File: tests/platform_test.js (hunk 3 above)
Comment (yurkovychv) [reply tests/platform_test.js:155]: faker has a bug actually. I can use another library for generating a password. more details [here](https://github.com/percona-platform/saas-ui/blob/main/cypress/pages/auth/getUser.ts#L4-L7)

File: tests/platform_test.js (hunk 2 above)
Comment (yurkovychv) [reply tests/platform_test.js:88]: Importing `core-ui` requires installing a lot of other dependencies. I'd prefer to add our own function here if you don't mind

File: tests/pages/perconaPlatform.js (hunk 1 above)
Comment (yurkovychv) [reply tests/pages/perconaPlatform.js]: fixed, thanks for pointing

**Review by puneet0191 — APPROVED** [review]

**Pull request merged**